### ISSUE TITLE
Issue Logging in to LG from HomeAssistant when setting up new integration

### ISSUE THREAD

**Issue by drfant** [ollo69/ha-smartthinq-sensors#320]
I just readded the Smart LG ThinQ sensors into HomeAssistant using HACS.  It seems to install correctly, but when I attempt to loginto LG it fails the login.  I have tried using the LOGIN screen in Homeassistant and also using the URL redirect.  I have reset the passwords on my LG account and am able to log into my account from my phone app, but not able to establish the connection from HomeAssistant.  Thoughts?

**Comment by ollo69** [comment]
Did you check that in phone apps there are new TOS that you have to accept?
If not, please attach here your HA log file.

**Comment by drfant** [comment]
Not sure what you mean by TOS.  This is the only info I have in the HA logs
2022-05-12 17:07:38 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] LGE ThinQ: Invalid Login info!
2022-05-12 17:09:02 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] LGE ThinQ: Invalid Login info!
2022-05-12 17:10:21 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] LGE ThinQ: Invalid Login info!
2022-05-12 18:01:55 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] LGE ThinQ: Invalid Login info!

I provide the login name, password, country code and language and hit submit.  I then get the error,  WHen I try with the URL redirect using I provide the credentials, and get a redirect URL and paste that in, but I also get an error at that time in HA.

**Comment by drfant** [comment]
Tried again using redirect URL and this is what I get:
2022-05-12 17:06:40 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration smartthinq_sensors which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-05-12 17:07:38 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] Error connecting to ThinQ: Authentication Error
2022-05-12 17:07:38 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] LGE ThinQ: Invalid Login info!
2022-05-12 17:09:02 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] Error connecting to ThinQ:
2022-05-12 17:09:02 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] LGE ThinQ: Invalid Login info!
2022-05-12 17:10:21 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] Error connecting to ThinQ:
2022-05-12 17:10:21 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] LGE ThinQ: Invalid Login info!
2022-05-12 18:01:55 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] Error connecting to ThinQ: Authentication Error
2022-05-12 18:01:55 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] LGE ThinQ: Invalid Login info!
2022-05-12 18:12:00 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] Error connecting to ThinQ: Authentication Error
2022-05-12 18:12:00 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] LGE ThinQ: Invalid Login info!
2022-05-12 18:13:01 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] Error connecting to ThinQ:
2022-05-12 18:13:01 ERROR (MainThread) [custom_components.smartthinq_sensors.config_flow] LGE ThinQ: Invalid Login info!


I have doubled checked my credentials and it still fails
